django_socio_grpc/protobuf/proto_classes.py

```diff
@@ -78,17 +80,24 @@ class ProtoField:
     """
 
     name: str
-    field_type: Union[str, "ProtoMessage"]
+    field_type: Union[str, "ProtoMessage", Enum]
     cardinality: FieldCardinality = FieldCardinality.NONE
     comments: list[str] | None = None
     index: int = 0
+    _field_type_str: str | None = None
```

Comment:
do you really need this extra attribute ? 
You can keep filed_type as enum until transforming it to str directly no ?

Reply:
because before field_type_str was only a property, before using the plugins i could just use the Enum name + ".Enum", but now the enum can be generated in a message or not, so it could just be the Enum name as well, and i only have that info in the generation plugin context.